reconcile/queries.py

```diff
@@ -309,12 +309,25 @@ def get_github_orgs():
       region
     }
     partition
+    {% if sharing %}
```

Comment:
I don't understand why this templating is needed, can you explain? It seems like `get_aws_accounts` could simply always return the sharing data and then in the integration since you're testing if ` sharing` is falsy (you could also check that length is > ?) the code would end up the same

Perhaps the only reason I can see is to make all the other `get_aws_accounts` graphql queries a little bit smaller and more efficient by not requesting unnecessary data?

Reply:
exactly. in the future, we may add more graphql objects under this section. this templating will make sure we do not need to update the allowed schemas list for other integrations.